### ISSUE TITLE
Update Launcher

### ISSUE THREAD

**Issue by gskleres** [Malow/MaloWLauncher#5]
We should add a button or a setting or something when it's started for the launcher to update itself if it sees a new version of itself is available.

**Comment by Malow** [comment]
Agreed. Currently the client will be informed about what the latest client version is and where it can download the latest version, and it will show it in a popup. But it would be really nice if it was handled automatically. I'm not quite sure how self-updating is solved usually, but I'm thinking having a small separate .exe file that is downloaded along with the .zip file for the latest version could work, where the MaloWLauncher launches this second .exe file and then closes itself, and the second .exe file deletes the MaloWLauncher.exe and replaces it with the downloaded one, and then launches MaloWLauncher.exe and closes itself.
